### ISSUE TITLE
No option for UK English

### ISSUE THREAD

**Issue by nathanielvirgo** [bartosz-antosik/vscode-spellright#494]
I am using OS X. Most applications allow me to set the language to "English (United Kingdom)", so clearly it exists as a system dictionary. However, in vscode-spellright it does not appear. If I try to select the language by clicking on "English" in the status bar, I get a list of languages that begins

 * English
 * Czech
 * Danish
 * German

If I search this list for "English", the only one that comes up is "English" - there seems to be no UK version available.

Is there a way that I can fix this issue? The spell checking is useful, but it's very annoying to have words underlined that are spelt correctly according to British conventions.

**Comment by martincook3** [comment]
Have a look at https://github.com/bartosz-antosik/vscode-spellright#settings. I think the `spellright.groupDictionaries` option will help you.

**Comment by chmac** [comment]
True, disabling the **Group dictionaries** option allows for the selection of British English.

@nathanielvirgo Open to closing this issue with this solution?

**Comment by markbattistella** [comment]
@nathanielvirgo This is an example with images: https://github.com/bartosz-antosik/vscode-spellright/issues/462#issuecomment-1144268701
